app/views/liquids/_user_subscription.html.erb

```diff
@@ -1,81 +1,66 @@
-<div class="ltag__user-subscription-tag" id="user-subscription-tag" data-author-username=<%= author_username %>>
+<div class="ltag__user-subscription-tag" data-author-username=<%= author_username %>>
   <div class="ltag__user-subscription-tag__container">
 
-    <div class="ltag__user-subscription-tag__content w-100" id="content">
+    <div class="ltag__user-subscription-tag__content w-100">
 
-      <div class="ltag__user-subscription-tag__profile-images signed-out" id="profile-images">
+      <div class="ltag__user-subscription-tag__profile-images signed-out">
 
-        <span class="crayons-avatar crayons-avatar--xl ltag__user-subscription-tag__author-profile-image">
-          <img class="crayons-avatar__image ltag__user-subscription-tag__author-profile-image" src=<%= author_profile_image %> <%= "#{author_username} profile image" %>>
+        <span class="crayons-avatar crayons-avatar--xl ltag__user-subscription-tag__author-profile-image m-auto">
+          <img class="crayons-avatar__image ltag__user-subscription-tag__author-profile-image m-0" src=<%= author_profile_image %> <%= author_username %> <%= author_username %>>
         </span>
 
-        <span class="crayons-avatar crayons-avatar--xl ltag__user-subscription-tag__subscriber-profile-image">
-          <img class="crayons-avatar__image ltag__user-subscription-tag__subscriber-profile-image">
+        <span class="crayons-avatar crayons-avatar--xl ltag__user-subscription-tag__subscriber-profile-image m-auto">
+          <img class="crayons-avatar__image ltag__user-subscription-tag__subscriber-profile-image m-0" alt="">
         </span>
 
       </div>
 
-      <h1 class="ltag__user-subscription-tag__cta-text fs-xl mt-0 mb-4 align-center" id="user-subscription-cta-text">
+      <h2 class="ltag__user-subscription-tag__cta-text fs-xl mt-0 mb-4 align-center">
         <%= cta_text %>
-      </h1>
+      </h2>
 
-      <div class="ltag__user-subscription-tag__subscription-area align-center" id="subscription-area">
-        <div class="ltag__user-subscription-tag__signed-out" id="subscription-signed-out">
-          <div class="fs-base mb-2" id="logged-out-text">
-            <%= t("views.liquids.user_subscription.logged_out") %>
+      <div class="ltag__user-subscription-tag__subscription-area align-center">
+        <div class="ltag__user-subscription-tag__signed-out">
+          <div class="fs-base mb-2">
+            <%= t("views.liquids.user_subscription.logged_out", community: community_name) %>
           </div>
-          <button class="crayons-btn" id="sign-in-btn">
+          <a href="/enter" class="c-cta c-cta--default">
```

Comment:
Previously this was a button that triggered the login modal. I've changed it to a CTA link, as I can't see any reason why we'd want the user to click "Sign in" only to have to click "Sign in" again on the modal 🤷‍♀️